### ISSUE TITLE
Right click menu is lost when title bar is moved onto top panel

### ISSUE THREAD

**Issue by Coeur-Noir** [franglais125/no-title-bar#65]
Hello,

when window is maximized, there is no longer access to the « move window » menu on right click : 
![right_click_on_title_bar](https://user-images.githubusercontent.com/1895947/38417238-dd779302-3997-11e8-86fd-d3e24bec2ec2.png)

**Comment by franglais125** [comment]
Yeah, I'd love to fix this. I'll get to it when I get some time! Meanwhile, PRs are welcome!
